docs/qe-available-tasks/sealights/python-instrumentation/python-instrumentation.md

```diff
@@ -0,0 +1,108 @@
+# Sealights Python Instrumentation Tekton Task
+
+This Tekton task automates the process of instrumenting python code with Sealights for quality analytics and testing. It retrieves the source code from a trusted artifact, installs Python Sealights agent, configures the app for sealights using vars from your pipeline run, scans all .py files that are not in the --exclude step above and reports scan to sealights, and stores results to be used later on in testing. The task can be triggered by either Pull Request or other events.
+
+## Overview
+
+This task performs the following steps:
+
+1. **Retrieves** the source code from a trusted artifact.
+2. **Configures & Scans** the Python application using Sealights.
+
+The task can be triggered by different events (e.g., Pull Request, Push) and allows users to exclude specific Python packages from the configuration process.
+
+## Parameters
+
+| Name                  | Type     | Default       | Description                                                                                   |
+|-----------------------|----------|---------------|-----------------------------------------------------------------------------------------------|
+| `source-artifact`     | `string` | -             | The Trusted Artifact URI pointing to the source code.                                         |
+| `python-version`          | `string` | -             | The Python version to use with the 'ubi8/python' image, in the format (e.g., '311').                                                       |
+| `sealights-secret`    | `string` | -             | The name of the OpenShift secret containing Sealights credentials.                            |
```

Comment:
should we mention what secret key name is expected?

Reply:
hmm I'm now thinking that this var isn't actually needed, I'll need to check this.

Reply:
hmm, I thought you need a token to send the data to Sealights when scanning

Reply:
Ya we do we are just getting the token from injecting a volume we don't need to pass it as a param like I had hear. I just tested the removal of this param and everything works fine.